scalene/scalene_profiler.py

```diff
@@ -2070,7 +2072,14 @@ def run_profiler(
                     # Splice in the name of the file being executed instead of the profiler.
                     the_globals["__file__"] = os.path.abspath(progs[0])
                     # Some mysterious module foo to make this work the same with -m as with `scalene`.
-                    the_globals["__spec__"] = None
+                    # the_globals["__spec__"] = spec
```

Comment:
Remove comment cruft - I'm not thrilled with "mysterious module foo" as a comment :). Can you point to any resource here, or elaborate on what's going on?